### PR TITLE
Add balances as a row in the UI vs just json

### PR DESCRIPTION
**Pull request by msfeldstein**
Add UI rows for each balance in the wallet, and add a send button so you can directly send that asset. Also clean up the code in makePayment.ts to use explicit naming vs an array of values.

Screenshot 2020-06-11 at 4 49 29 PM

### CODE REVIEW

**Comment by tyvdh** [comment]
```
% npm start

> stellar-demo-wallet@0.0.46 start /Users/tylervanderhoeven/Desktop/Web/Stellar/stellar-demo-wallet
> stencil build --dev --watch --serve

[47:38.5]  @stencil/core v1.9.2 🚦
[47:38.7]  build, stellar-wallet, dev mode, started ...
[47:38.8]  transpile started ...
[47:40.1]  transpile finished in 1.36 s
[47:40.1]  type checking started ...
[47:40.1]  copy started ...
[47:40.1]  generate styles started ...
[47:40.1]  bundling components started ...
[47:40.2]  copy finished (0 files) in 39 ms
[47:40.5]  generate styles finished in 355 ms

[ ERROR ]  Rollup: Unresolved Import
           Could not resolve './events/views/balanceDisplay' from
           src/components/wallet/lazy_yvonwnmq7ztsqo8jbesgq-wallet.js

[47:40.5]  build failed, watching for changes... in 1.82 s

[47:42.9]  build, stellar-wallet, dev mode, started ...

[ ERROR ]  TypeScript: src/components/wallet/wallet.ts:4:28
           Cannot find module './events/views/balanceDisplay'.

      L4:  import balanceDisplay from './events/views/balanceDisplay'

[47:42.9]  build failed, watching for changes... in 1 ms
```

Looks like the `./events/views/balanceDisplay` wasn't committed?

**Comment by msfeldstein** [comment]
Ooopsie, updated

File: src/components/wallet/views/balanceDisplay.tsx [hunk 1]
```diff
@@ -9,21 +9,26 @@ interface Balance {
...
-    const name = balance.asset_type === 'native' ? 'XLM' : balance.asset_code
+    const assetCode =
+      balance.asset_type === 'native' ? 'XLM' : balance.asset_code
+
     return (
       <div class="balance-row">
-        <div class="asset-code">{name}</div>
-        <div class="balance">{parseFloat(balance.balance).toFixed(2)}</div>
+        <div class="asset-code">{assetCode}</div>
+        <div class="balance">{balance.balance}</div>
```
Comment (tyvdh) [inline src/components/wallet/views/balanceDisplay.tsx:20]: Feels pretty important to include all decimals. We could try and be smarter with the info from TOML files but for now it feels right to include whatever is returned.

Comment (msfeldstein) [reply src/components/wallet/views/balanceDisplay.tsx:20]: agreed i'll change

Comment (msfeldstein) [reply src/components/wallet/views/balanceDisplay.tsx:20]: oh you did change, 👍

File: src/components/wallet/methods/updateAccount.ts [hunk 2]
```diff
@@ -12,31 +15,32 @@ export default async function updateAccount() {
...
             'account_id',
-            'signers',
+            // 'balances',
+            // 'signers',
             'paging_token',
-            'data_attr',
+            // 'data_attr',
           ]),
         }
       })
```
Comment (tyvdh) [inline src/components/wallet/methods/updateAccount.ts:46]: Do these changes feel right? Would like to nuke the comments but unsure if this is the right list for the raw output.

Comment (msfeldstein) [reply src/components/wallet/methods/updateAccount.ts:46]: yeah i think these are a good set of properties to keep.  What is data_attr?  Definitely should omit _links.

Comment (msfeldstein) [reply src/components/wallet/methods/updateAccount.ts:46]: i'll push

Comment (tyvdh) [reply src/components/wallet/methods/updateAccount.ts:27]: `data_attr` is the managed data on an account:
https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-data

**Pull request merged**